Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>What's the example? We can just put "Did specific user sign-up".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Updated on push to prob of rain</p>

Reply:
<p>I don't think this is right. The outcome is on the x-axis, so here it would be rain yes or no and then probability is on the y-axis.</p>

Reply:
Fixed